diff --git a/20220301/common_command.sh b/20220301/common_command.sh
index c217f1d..7b1a169 100644
--- a/20220301/common_command.sh
+++ b/20220301/common_command.sh
@@ -51,15 +51,15 @@ function GET_COMMAND(){
     elif [ "${UNAME}" == "mattshamrock" ]; then
 
 	# mattshamrockさん用のtetris
-	if [ "${LEVEL}" == "2" ]; then
+#	if [ "${LEVEL}" == "2" ]; then
 	    # level2
 	    ADD_COMMAND="curl -LJO https://raw.githubusercontent.com/mattshamrock/tetris2/main/block_controller.py &&\
 	    mv block_controller.py game_manager/."
-	else
-	    # level3
-	    ADD_COMMAND="curl -LJO https://raw.githubusercontent.com/mattshamrock/tetris3/main/block_controller.py &&\
-	    mv block_controller.py game_manager/."
-	fi
+#	else
+#	    # level3
+#	    ADD_COMMAND="curl -LJO https://raw.githubusercontent.com/mattshamrock/tetris3/main/block_controller.py &&\
+#	    mv block_controller.py game_manager/."
+#	fi
 	EXEC_COMMAND="${ADD_COMMAND} && python3 start.py -l ${LEVEL} -d ${DROP_SPEED} -t ${GAME_TIME} -r ${RANDOM_SEED} -u ${UNAME} -f ${LOGFILE} -m predict"
 	
     else
diff --git a/20220301/tournament.sh b/20220301/tournament.sh
index ceb2e5f..8e1774b 100755
--- a/20220301/tournament.sh
+++ b/20220301/tournament.sh
@@ -57,7 +57,8 @@ function do_game(){
     #local PROGRAM_NAME="sample_program"
 
     # sound name
-    local SOUND_NUMBER=`echo $(( $[SOUND_NUMBER] % ${#SOUNDFILE_LIST[@]} ))`
+    local SOUND_NUMBER=`echo $((RANDOM%+3))` # 0-2 random value
+#    local SOUND_NUMBER=`echo $(( $[SOUND_NUMBER] % ${#SOUNDFILE_LIST[@]} ))`
     local SOUNDFILE_PATH=${SOUNDFILE_LIST[$SOUND_NUMBER]}
 
     # prepare
@@ -168,15 +169,15 @@ function do_game_main(){
     echo -n >| ${RESULT_LOG}
 
     ## sample
-    PLAYER1="seigot@master@せいご-program"
-    PLAYER2="isshy-you@master@isshy-program"
+#    PLAYER1="seigot@master@せいご-program"
+#    PLAYER2="isshy-you@master@isshy-program"
     
     ## 2
-#    PLAYER1="yuin0@tetris_second@CrackedEgg_v1.9"
+    PLAYER1="yuin0@tetris_second@CrackedEgg_v1.9"
 #    PLAYER1="isshy-you@ish05h3@いっしー5号ぷらす"
 #    PLAYER2="4321623@v2.0@勇者ちゃん2号"
-#    PLAYER1="usamin24@Lv2@チョコ&レート2号改"
-#    PLAYER2="mattshamrock@master@高まるフォイ"
+#    PLAYER2="usamin24@Lv2@チョコ&レート2号改"
+    PLAYER2="mattshamrock@master@高まるフォイ"
 
 
     ## 2_ai
@@ -187,13 +188,13 @@ function do_game_main(){
     ## 3
 #    PLAYER1="bushio@submit_level3@AIでテトリス"
 #    PLAYER1="usamin24@Lv3@チョコ&レート3号"
-#    PLAYER2="mattshamrock@master@困るフォイ"
-#    PLAYER1="isshy-you@ish05h3@いっしー5号ぷらす"
+#    PLAYER1="mattshamrock@master@困るフォイ"
+#    PLAYER2="isshy-you@ish05h3@いっしー5号ぷらす"
 #    PLAYER2="yuin0@tetris_second@CrackedEgg_v1.9"
 
     ## ryuo
 #    PLAYER1="bushio@submit_level3@AIでテトリス"
-#    PLAYER1="isshy-you@ish05c@いっしー5号"
+#    PLAYER2="isshy-you@ish05c@いっしー5号"
 #    PLAYER2="usamin24@Lv3@チョコ&レート3号"
 
     #---
